### PR TITLE
add breakpointts from aries theme and pad for page

### PR DESCRIPTION
**Pull request by britt6612**
#### What does this PR do?
adds new breakpoints from aries site
#### What testing has been done on this PR?
all of this was locally within design system site
#### Is this change backward compatible or could it be a breaking change for the official HPE theme?
breaking change of break points
#### How should this PR be communicated in the release notes?
announcement with migration notes

### CODE REVIEW

File: src/js/themes/hpe.js [hunk 1]
```diff
@@ -27,6 +29,47 @@ const deepFreeze = (obj) => {
 export const hpe = deepFreeze({
   defaultMode: 'light',
   global: {
+    breakpoints: {
```
Comment (taysea) [inline src/js/themes/hpe.js:32]: Thank you!

File: src/js/themes/hpe.js [hunk 2]
```diff
@@ -910,17 +953,23 @@ export const hpe = deepFreeze({
...
       width: {
         min: '336px', // 336 + 24 (margin) + 24 (margin) = 384 (e.g. 'medium')
         max: 'large',
       },
-    },
-    full: {
-      width: {
-        min: '336px', // 336 + 24 (margin) + 24 (margin) = 384 (e.g. 'medium')
-        max: '100%',
+      xsmall: {
```
[the remark this answers is not in the document]
Comment (britt6612) [reply src/js/themes/hpe.js:968]: yoou are right I didnt mean to delete this

[the remark this answers is not in the document]
Comment (britt6612) [reply src/js/themes/hpe.js:968]: we might of been using grommet base for that let me confirm

[the remark this answers is not in the document]
Comment (britt6612) [reply src/js/themes/hpe.js:968]: added these

[the remark this answers is not in the document]
Comment (taysea) [reply src/js/themes/hpe.js:968]: I'm not seeing them added, you might need to push up once more. I believe we need to add:

```
 full: {
      xsmall: {
        pad: { horizontal: 'large' },
      },
      xlarge: {
        pad: { horizontal: 'large' },
      },
    },
```
I tested locally on DS site  by switching layout to `Page kind="full"`. Without adding the "full" modifications, everything touches the edges of the page (incorrect):
Screen Shot 2022-03-31 at 11 17 18 PM

But after adding it, it looks has padding:
Screen Shot 2022-03-31 at 11 17 39 PM

[the remark this answers is not in the document]
Comment (britt6612) [reply src/js/themes/hpe.js:968]: must of been tired I commited but forgot to push!

**Review by taysea — APPROVED** [review]

**Review by halocline — APPROVED** [review]

**Review by jcfilben — APPROVED** [review]

**Pull request merged**